### PR TITLE
Adding gps_altitude field in class packet

### PR DESCRIPTION
**Pull request by sanujkul**

### CODE REVIEW

File: packet.cpp [hunk 1]
```diff
@@ -33,6 +34,7 @@ String packet::toString(){
 		this->gps_time + "," +
```
Comment (Shikhar1998) [inline packet.cpp]: This can be changed to `String(this->gps_time)` as this is returned as 'int32'.

File: packet.h [hunk 2]
```diff
@@ -27,6 +27,7 @@ class packet {
 	String gps_time;			// the time generated by the GPS receiver. The time must be reported in UTC and have a resolution of a second. UTC Format: MM/dd/yyyy KK:mm:ss a
```
Comment (Shikhar1998) [inline packet.h]: Similarly, a 'volatile' variable can be created below as is returned as int32. Rest is good.

File: packet.cpp (hunk 1 above)
Comment (sanujkul) [reply packet.cpp]: Sir, do I have to make changes in file and push them again or changes you've made are automatcally added to file?  I'm new to GitHub! and same doubt for second comment of adding 'volatile' to gps_time

**Comment by Shikhar1998** [comment]
Just make the changes, and select this pull request (or this branch), after making the modifications. If you wish to open another pull request, make all the changes presently in this pull request plus all those which I have suggested. You have the liberty to make some errors while adding code as you are still learning, nothing to worry about :).

**Comment by sanujkul** [comment]
> Hey, don't close it. All the changes are clubbed, you did it right.

Sir, I've made changes you've suggested and pushed it in this branch. Do I have to make another pull request?

**Comment by Shikhar1998** [comment]
When, you merge the branch is merged to master, and becomes the new master. You don't need an extra pull request. Cool then, good.

**Comment by sanujkul** [comment]
> When, you merge the branch is merged to master, and becomes the new master. You don't need an extra pull request. Cool then, good.

Thank you Sir! :)

**Pull request merged**